decidim-api/spec/types/mutation_type_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+require "spec_helper"
+require "decidim/api/test/type_context"
+
+module Decidim
+  module Api
+    describe MutationType do
+      include_context "graphql type"
+
+      describe "addComment" do
+        let!(:participatory_process) { create(:participatory_process, organization: current_organization) }
+
+        let(:query) { 
+          "{ addComment(commentableId: \"#{participatory_process.id}\", commentableType: \"Decidim::ParticipatoryProcess\", body: \"This is a new comment\") { id, body } }" 
```

Comment:
I'd rather have an `addProcessComments(id, body)`, or an `addComments` scoped to a process(it that exists in GraphQL)